### PR TITLE
Genericboot

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**

### CODE REVIEW

File: core/arch/arm/plat-stm/core_bootcfg.c [hunk 1]
```diff
@@ -24,193 +24,7 @@
...
-#include <platform_config.h>
-#include <mm/core_mmu.h>
-#include <mm/core_memprot.h>
-#include <util.h>
-#include <kernel/tee_misc.h>
-#include <trace.h>
-
 /*
- * define the platform memory Secure layout
+ * This is a tempory step going to the generic boot scheme
```
Comment (jbech-linaro) [inline core/arch/arm/plat-stm/core_bootcfg.c]: Spelling: "temporary"

**Comment by pascal-brand-st-dev** [comment]
With the missing push of the change in generic_core_bootcfg.c

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by pascal-brand-st-dev** [comment]
@jforissier Is it ok for you?

**Comment by jforissier** [comment]
@pascal-brand-st-dev yes

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Pull request merged**